kvpress/attention_patch.py

```diff
@@ -0,0 +1,57 @@
+import torch
+from transformers.modeling_utils import ALL_ATTENTION_FUNCTIONS
+
+
+def search_hyperplane(X, max_iter: int = 1000):
+    """
+    Search for an hyperplane Y such that for every Xi, <Xi, Y> <= 0
```

Comment:
Could you add the shape of X and Y in the docstring (indicating it is a 3d tensor), and maybe add a test for the function?